model/fv_dynamics.F90

```diff
@@ -521,9 +535,16 @@ subroutine fv_dynamics(npx, npy, npz, nq_tot,  ng, bdt, consv_te, fill,
 !            call Ray_fast(abs(dt), npx, npy, npz, pfull, flagstruct%tau, u, v, w,  &
 !                          dp_ref, ptop, hydrostatic, flagstruct%rf_cutoff, bd)
 !         else
-             call Rayleigh_Super(abs(bdt), npx, npy, npz, ks, pfull, phis, flagstruct%tau, u, v, w, pt,  &
+             call Rayleigh_Super(abs(bdt), npx, npy, npz, ks, pfull, phis, flagstruct%tau, flagstruct%tau_w, u, v, w, pt,  &
+#ifdef MULTI_GASES
+                  q, ncnst,  &
+#endif
```

Comment:
Why insert this in the middle of the argument in the list instead of at the end?

Reply:
I did not change this as many "multi_gases" are doing this way.